### ISSUE TITLE
About Redux and express the router to config

### ISSUE THREAD

**Issue by whevether** [coryhouse/pluralsight-redux-starter#26]
Node version:4.5

npm version:2.15.9

Operating system: windows 7

How are you doing. I would like to ask God to you. How to use Express and react after the framework using the router;

**Comment by coryhouse** [comment]
Hi, sorry, I don't understand your question. Can you rephrase it?

**Comment by whevether** [comment]
**Hello, I am a Chinese reader, English is not very good; I would like to ask is that when using express route services how to write:
This is my client routing:**
export default (
    <Route name="app" path="/" component={App}>
        <IndexRoute component={HomePage} />
        <Route path="reddit" component={RedditPage} />
        <Route path="todo" component={TodoPage} />
        <Route path="counter" component={counterPage} />
        <Route path="about" component={AboutPage} />
        <Route path="form1" component={ReduxFormSimple}/>
        <Route path="form2" component={ReduxSyncFormValidation}/>
        <Route path="form3" component={ReduxSubmitFormValidation}/>
        <Route path="form4" component={AsyncValidateForm}/>
        <Route path="form5" component={InitialStateForm}/>
        <Route path="form6" component={FieldFormArray}/>
        <Route path="form7" component={FieldNormailizeFomrPage}/>
        <Route path="form8" component={ImmuTableFormPage}/>
        <Route path="form9" component={SelectFormValuesPage}/>
        <Route path="form10" component={WizardFormPage}/>
        <Route path='*' component={error404}/>
    </Route>
);
2:**Express route services rendering is written like this do?**   
import express from 'express';
import React from 'react';
import {RouteContext,match} from 'react-router';
import routes from '../app/routers/routes';
const app = express();

app.get('/', function (req, res) {
    match.match({routes,location: req.url }, (error, redirectLocation, renderProps) =>{
       console.log(req.url);
        console.log(routes);
    });
});

**Comment by injune1123** [comment]
Hi, @whevether . I am also a Chinese coder, and I am happy to help. Can you describe your problem in Chinese?你能用中文说说你有什么问题吗？或者贴个链接？

**Comment by whevether** [comment]
@injune1123   你好我已经解决了。我之前是想问express 框架服务端渲染react的问题的。  谢谢你哈.

**Comment by injune1123** [comment]
解决了就好，不客气。

**Comment by coryhouse** [comment]
Closing since I can't read Chinese, but it appears the conversation has stopped.
